frontend/src/pages/LocationPage.tsx

```diff
@@ -40,7 +37,9 @@ const useStyles = makeStyles({
   },
 });
 
-const LocationPage = ({ data }: Props): ReactElement => {
+const LocationPage = (): ReactElement => {
+  const [homeData, setHomeData] = useState<CardData[]>([]);
```

Comment:
This variable should not be named `homeData` as it's the one for data in homepage. It would be better if you name it `aptCards` or just `data`